brave-spancollector-kafka/src/main/java/com/github/kristofa/brave/kafka/KafkaSpanCollector.java

```diff
@@ -91,6 +100,7 @@ public void close() {
         }
         executorService.shutdown();
         producer.close();
+        metricsHandler.incrementDroppedSpans(queue.size());
```

Comment:
I think what we have is correct because we wait for `SpanProcessingTask` to finish before calling `shutdown`. So I expect that after `shutdown` is called `isTerminated` should be true. 

In other words, once `SpanProcesssingTask` is finished the remaining items on the queue won't be processed anymore.